client/js/components/statement/assessmentTable/DpFragmentsSwitcher.vue

```diff
@@ -168,6 +170,10 @@ export default {
     showAllFragments () {
       this.allFragmentsShown = !this.allFragmentsShown
       this.$emit('fragments:showall', this.allFragmentsShown)
+    },
+
+    toggle () {
+      this.$emit('input', (this.statementTabVisible === false))
```

Comment:
This has to be changed to

```suggestion
      this.$emit('toggletabs', (this.statementTabVisible === false))
```